### ISSUE TITLE
npm audit found an high problem with this package

### ISSUE THREAD

**Issue by frederikhors** [onderceylan/pwa-asset-generator#810]
**Describe the bug**
npm audit found an high problem with this package

**To Reproduce**
1. `npm i`
2. read the message in console:

```
# npm audit report

node-fetch  <2.6.7
Severity: high
node-fetch is vulnerable to Exposure of Sensitive Information to an Unauthorized Actor - https://github.com/advisories/GHSA-r683-j2x4-v87g
fix available via `npm audit fix --force`
Will install pwa-asset-generator@4.3.2, which is a breaking change
node_modules/puppeteer-core/node_modules/node-fetch
  puppeteer-core  10.0.0 - 13.1.1
  Depends on vulnerable versions of node-fetch
  node_modules/puppeteer-core
    pwa-asset-generator  >=4.3.3
    Depends on vulnerable versions of puppeteer-core
    node_modules/pwa-asset-generator

3 high severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force
```

**Expected behavior**
To fix this.

**System (please complete the following information):**
 - OS: win 10
 - node version: 16
 - npm version: 8
 - cli version: 6.0.6

**Comment by onderceylan** [comment]
Hi @frederikhors, thanks for reporting! I'm looking into it.

**Comment by onderceylan** [comment]
Fixed with version 6.0.7!
https://github.com/onderceylan/pwa-asset-generator/releases/tag/v6.0.7
